### ISSUE TITLE
Set up a curse page as soon as the first usable build is ready

### ISSUE THREAD

**Issue by Sunconure11** [ljfa-ag/Totemic#6]
Even though this is probably already planned.

**Comment by ljfa-ag** [comment]
Yup, I'm going to do that. I want to make sure everything that is there is working without major bugs, and I need to write more Totempedia entries.
